v2/pkg/protocols/network/network_test.go

```diff
@@ -17,7 +17,7 @@ func TestNetworkCompileMake(t *testing.T) {
 	templateID := "testing-network"
 	request := &Request{
 		ID:       templateID,
-		Address:  []string{"{{Hostname}}", "{{Hostname}}:8082", "tls://{{Hostname}}:443"},
+		Address:  []string{"tls://{{Hostname}}:443"},
```

Comment:
Isn’t `{{Host}}:443` is expected as `{{Hostname}}` already includes port information?

Reply:
Shouldn't matter as the variables aren't replaced at request compile time, but `{{Host}}` would be more consistent